client/src/Client/pages/Passenger/OrderRide/OrderRide.tsx

```diff
@@ -248,30 +248,30 @@ const OrderRide = () => {
               יש לבחור מספר נוסעים
             </FormHelperText>
           )}
-        </FormControl>
+          </FormControl>
         <FormControl>
           <TextField
-            label={user ? 'הערה' : 'מטרת הנסיעה'}
+            label={user && 'תיאור הנסיעה'}
```

Comment:
@aviv-taori 
I have no idea why this was different for with `user` or without.
But for sure it does not make sense to return `null|undefined` when user does not exist.
I think you should simply remove the `user &&`.

Reply:
If you think there is no meaning to the "user", I will delete..

Reply:
@aviv-taori 

There is meaning, I'm just not sure it's the meaning you want.
What `user && ` means is that only when a connected user exists you will get the label `תיאור הנסיעה`, otherwise you will see no label.
This will cause that you won't have a label on guest mode / war mode, which you probably are in.

What was happening before for some unknown reason is that when you are in guest mode you will see `הערה` and when you are in a connected user mode you will see `מטרת הנסיעה`.

What you basically have done here is removed the label for all guest rides. Is that what you intended to do?

Reply:
@LEVI-RIVKIN 

I wasn't sure what "user" meant.
I thought as long as there is a ride - 'user', thank you very much for the comment.

 In any case, should I update as long as there is a trip with a 'תיאור הנסיעה' tag? @OrleeTal

Reply:
@aviv-taori What do you mean by "as long as there is a trip"?
Let's have a google meet and finish with this.. LMK when you're available.